mongodb/intro/date-aggregation/aggregation-with-date-expression-operators-iii.md

```diff
@@ -0,0 +1,175 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+  - basic
+  
+aspects:
+  - introduction
+  - new
+
+type: normal
+
+category: how to
+
+---
+
+# Aggregation With $hour, $minutes, $seconds and $milliseconds
+
+---
+## Content
+
+For the document used in aggregation, make sure to check the Previous Document[1] footnote.
+
+### $hour
+
+The `$hour` date operator is used to extract the exact hour the document was created on. The output goes from 0-23.
+
+Example:
+```js
+db.pokedex.aggregate(
+   [
+     {$match: {"name": "Bulbasaur"}},
+     {
+       $project:
+         {
+           Hour: { $hour: "$Date" },
+           _id: 0
+         }
+     }
+   ]
+)
+```
+
+Output:
+```js
+{ 
+  "Hour" : 7 
+}
+```
+
+### $minutes
+
+The `$minute` date operator is used to extract the exact minute the document was made on. The output goes from 0-59.
+
+Example:
+```js
+db.pokedex.aggregate(
+   [
+     {$match: {"name": "Bulbasaur"}},
+     {
+       $project:
+         {
+           Minutes: { $minute: "$Date" },
+           _id: 0
+         }
+     }
+   ]
+)
+```
+Output:
+```js
+{ 
+  "Minutes" : 21 
+}
+```
+
+### $seconds
+
+The `$seconds` date operator is used to extract the exact second the document was made on. The output goes from 0-59, however it is possible for the output to be 60 in case of a leap second.
```

Comment:
maybe add a short footnote explaining what a leap second means?
```suggestion
The `$seconds` date operator is used to extract the exact second the document was made on. Although the output goes from 0 to 59, it is possible for the output to be 60 in case of a leap second.
```